Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -4718,6 +4718,73 @@ def panorama_add_static_route_command():
     })
 
 
+def panorama_override_vulnerability(threatid: str, vulnerability_profile: str, drop_mode: str):
+    xpath = "{}profiles/vulnerability/entry[@name='{}']/threat-exception/entry[@name='{}']/action".format(
+        XPATH_OBJECTS,
+        vulnerability_profile,
+        threatid)
+    params = {'action': 'set',
+              'type': 'config',
+              'xpath': xpath,
+              'key': API_KEY,
+              'element': "<{0}></{0}>".format(drop_mode)
+              }
+
+    return http_request(
+        URL,
+        'POST',
+        body=params,
+    )
+
+
+@logger
+def panorama_get_predefined_threats_list(target: str):
+    """
+    Get the entire list of predefined threats as a file in Demisto
+    """
+    params = {
+        'type': 'op',
+        'cmd': '<show><predefined><xpath>/predefined/threats</xpath></predefined></show>',
+        'target': target,
+        'key': API_KEY
+    }
+    result = http_request(
+        URL,
+        'GET',
+        params=params
+    )
+    return result
+
+
+def panorama_get_predefined_threats_list_command():
+    target = str(demisto.args()['target']) if 'target' in demisto.args() else None
```

Comment:
why not just use get? dose it make a difference to put here None and not ''?
```suggestion
    target = str(demisto.args().get('target',''))
```

Reply:
Yes. it needs to be a None if not existing and this is the convention already in the integration code.

Reply:
example:
![Screen Shot 2020-07-07 at 9 30 29](https://user-images.githubusercontent.com/37335599/86727906-89be3d80-c034-11ea-9384-1d51fe61e0b0.png)